### ISSUE TITLE
Why trained on weakly supervised setting?

### ISSUE THREAD

**Issue by winston52** [wenwenmin/scstGCN#3]
Hi Authors, 

Thank you for sharing the code and the data. From your paper: 

> We adopt a weakly supervised learning framework to train scstGCN because the model outputs are at the superpixel-level while the training data are at the spot-level. Specifically, we model the sum of gene expression of the superpixels covered by spots and the gene expression of spots.

I also checked your code and noticed that you did not use the high-resolution data during training, but you did use it during evaluation.

I am a bit confused about why you trained your model in a weakly supervised setting. Since we can obtain ground truth high-resolution data and pseudo spot-level low-resolution data from the Xenium data or VisiumHD data through preprocessing, why did you choose to use the low-resolution data as the label to compute the loss rather than the high-resolution data? The training goal is not consistant with the evluation goal. 

Could you share some clarification about this? Thank you.

**Comment by shuailinxue** [comment]
@winston52 The goal of scstGCN is to enhance the resolution of spatial transcriptomics data based on sequencing technologies (such as ST and Visium) to achieve a resolution close to the single-cell level. Xenium and Visium HD have super-resolution gene expression as ground truth, but they cannot be involved in the training process because ST and Visium do not have super-resolution gene expression (The real goal). In other words, the simulation experiments conducted on Xenium and Visium HD are solely for evaluating the model's accuracy.

**Comment by winston52** [comment]
@shuailinxue Thank you for your clarification.
